modular-docs-manual/content/topics/using_text_snippets_or_text_fragments.adoc

```diff
@@ -17,13 +17,16 @@
 [NOTE]
 The following standard is recommended when the documentation is being maintained without a Content Management System (CMS) capable of managing complex interrelations between modules.
 
-The use of reusable text snippet files (or text fragment files) is discouraged due to the complications that can arise due to the complexity they introduce.
-
-Snippet (fragment) file use should be limited to:
+Use reusable text snippet files (or text fragment files) only for the following:
 
 * Standardized admonitions (such as 'Technology preview' and 'Beta' text).
 * Where there is an existing standard between the upstream and downstream communities.
```

Comment:
What would this be, specifically?

Reply:
"Where there is an existing standard between the upstream and downstream communities." clause also comes from the original text. I can find out what the specifics were.